src/app/clusters/network-commissioning/network-commissioning.cpp

```diff
@@ -767,6 +767,53 @@ void Instance::HandleAddOrUpdateWiFiNetworkWithPDC(HandlerContext & ctx,
 }
 #endif // CHIP_DEVICE_CONFIG_ENABLE_WIFI_PDC
 
+void Instance::OnThreadCredentialsAvailable(chip::ByteSpan *     pOperationalDataset,
+                                            Optional<uint64_t> * pAddOrUpdateThreadNetworkBreadcrumb,
+                                            chip::ByteSpan *     pNetworkID,
+                                            Optional<uint64_t> * pConnectNetworkBreadcrumb)
+{
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
```

Comment:
how about making the whole method be depending on this enabling, not just the implementation?